src/Main/Gear.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from 'common/Icon';
+
+class Gear extends React.PureComponent {
+  static propTypes = {
+    selectedCombatant: PropTypes.object.isRequired,
+  };
+  render() {
+    const toolTipBaseUrl='http://www.wowhead.com/item=';
+    const data = Object.values(this.props.selectedCombatant._gearItemsBySlotId);
+    const getBorderColor = (quality) => { // Not used right now
+      switch (quality) {
+        case 6:
+          return '#e5cc80';
+        case 5:
+          return '#ff8000';
+        case 4:
+          return '#a335ee';
+        case 3:
+          return '#0070dd';
+        case 2:
+          return '#1eff00';
+        case 1:
+          return '#ffffff';
+        default:
+          break;
+      }
+    };
+    const artifactData = data.filter(item => item.quality === 6 && item.gems); // get artifact from list of items to render relics
```

Comment:
This kinda qualifies as a hack. You can achieve the same while using the array methods normally;
```jsx
const artifact = data.find(item => item.quality === 6);
const artifactData = artifact ? artifact.gems : null;
```
I think it's safe to assume a player only has *one* artifact.

Also even more nitpicky: the var name `artifactData` is a bit vague. How about `relics`?